app/shared/compact-reservation-list/CompactReservationList.spec.js

```diff
@@ -18,6 +19,7 @@ describe('shared/compact-reservation-list/CompactReservationList', () => {
     resources: Immutable({
       [resource.id]: resource,
     }),
+    onRemoveClick: simple.mock(),
```

Comment:
- [ ] Change this to just plain arrow function so we don't accidentally use the same mock in multiple tests.
```
onRemoveClick: () => null,
```